reference/pipeline/stages/index.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: single
+title:  "Pipeline Stages"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+> Note: this is a first pass at this documentation; more details will be added
+> soon.
+
+This article describes the currently-supported stages that you can add to your
+Spinnaker pipelines.
+
+## General
+
+### Bake
+Bakes an image in the specified region.
+
+### Canary Analysis
+Runs automated canary analysis against the deployment before fully deploying.
+For more information on automated canary analysis in Spinnaker, see the
+[user guide](/guides/user/canary/).
+
+### Check Preconditions
+Checks for preconditions before continuing. Specifically, you can either check
+that your cluster is a particular size, or add a pipeline expression. See the
+[pipeline expressions guide](/guides/user/pipeline-expressions/) for more
+information about creating and using pipeline expressions.
+
+### Clone Server Group
+Deploys a new Server Group which is a copy of the specified Server Group.
+
+### Deploy
+Deploys the previously baked or found image using the specified deployment
+strategy. Spinnaker provides built-in support for both red/black (also known as
+blue/green) and highlander deployment strategies. You can also choose to deploy
+with no impact on existing Server Groups, or build your own custom deployment
+strategy.
+
+### Destroy Server Group
+Deletes a Server Group and its resources from the specified Cluster. You must
+specify whether you want to delete the newest, oldest, or previous
+(second-most-recently deployed) Server Group when this stage starts.
+
+### Disable Cluster
+Disables the specified Cluster, which means that the cluster remains up but
+stops handling traffic. If desired, you can leave a specific number of Server
+Groups running while the rest of the cluster is disabled.
+
+### Disable Server Group
+Disables a Server Group, which means that the Server Group remains up but stops
+handling any traffic. This makes it easy to both route traffic to a new
+Server Group and roll back those changes if necessary. You must choose whether
+to disable the Server Group which is newest, oldest, or previous
+(second-most-recently deployed) when this stage starts.
+
+### Enable Server Group
+Enables a Server Group, which means that the Server Group will begin handling
+traffic again.
```

Comment:
"again" implies that it was enabled, then disabled, and this is now re-enabling. But would this stage ever be used to enable a server group *originally*?  I'm guessing no, but I just don't know.

Reply:
Oh, and if this is right--that it's *re*-enabling--you could shorten it:
s/will begin handling traffic again/will resume handling traffic/

Reply:
Yep, double checked and it seems like it's always re-enabling - will update this to use  your wording :)